invokeai/app/api/routers/app_info.py

```diff
@@ -54,6 +68,17 @@ async def get_version() -> AppVersion:
     return AppVersion(version=__version__)
 
 
+@app_router.get("/full_version", operation_id="app_full_version", status_code=200, response_model=AppFullVersion)
+async def get_full_version() -> AppVersion:
+    return AppFullVersion(
+        version=__version__,
+        python_version=sys.version,
+        torch_version=torch.version.__version__,
+        cuda_version=torch.version.cuda,
+        torchvision_version=torchvision.version.__version__,
+    )
```

Comment:
compel & compel don't have Pythonic ways to get the version.

Reply:
You can use builtin `importlib` to get the version of any package installed via pypi (which is all of the ones we use I believe)

```py
>>> from importlib.metadata import version
>>> version('compel')
'2.0.2'
```

Reply:
Also missing `xformers`, which, being an optional dependency, needs exception handling

Reply:
Hi @psychedelicious 

I didn't know `importlib.version`, thank you ! We learn everyday.

- I used it to import all version except pytorch because I need the CUDA version too.
- I added a 10s of other dependencies that I think important from their usage in invokeai
- I removerd the suffix `_version` from each field